pkg/model/hornet/hash.go

```diff
@@ -14,6 +14,18 @@ var (
 // Hash is the binary representation of a trinary Hash.
 type Hash []byte
 
+func HashFromAddressTrytes(trytes trinary.Trytes) Hash {
+	return trinary.MustTrytesToBytes(trytes[:81])[:49]
+}
+
+func HashFromHashTrytes(trytes trinary.Trytes) Hash {
+	return trinary.MustTrytesToBytes(trytes[:81])[:49]
+}
+
+func HashFromTagTrytes(trytes trinary.Trytes) Hash {
+	return trinary.MustTrytesToBytes(trytes)[:17]
```

Comment:
why not cut the input to 27?

Reply:
because I forgot how many trytes a tag has and then forgot to look it up